jenkins/Jenkinsfile.premerge

```
@@ -72,3 +78,24 @@ pipeline {
         }
     } // end of stages
 } // end of pipeline
+
+boolean isSignedOff() {
+    def target_rev = sh(returnStdout: true,
+            script: "git rev-parse refs/remotes/origin/${ghprbTargetBranch}").trim()
+    def revs_arr = sh(returnStdout: true,
+            script: "git log ${target_rev}..${ghprbActualCommit} --pretty=format:%h").split()
+
+    def signed_off = false
+    for( String commit : revs_arr ) {
+        def signed_log = sh(returnStdout: true,
+                script: "git show ${commit} --shortstat | grep 'Signed-off-by'")
```

I think there's an issue here where if `grep` fails to find a match it will return a non-zero exit code.  That in turn will cause the pipeline command to return a non-zero exit code which will cause the `sh` command to throw an exception as described [here](https://www.jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script).  I think this either needs to use the `returnStatus: true` option and switch off the result code of the step or do something like this to avoid `grep` failing to match to cause a pipeline failure:
```suggestion
                script: "git show ${commit} --shortstat | grep 'Signed-off-by || echo'")
```